native-script.rb

```diff
@@ -136,6 +136,7 @@ def install_environment_variable(name, value)
 # For more information see: https://github.com/CocoaPods/Xcodeproj/pull/393#issuecomment-231055159
 install("CocoaPods", "Installing CocoaPods... This might take some time, please, be patient.", 'gem install cocoapods -V', true)
 install("CocoaPods", "Setup CocoaPods... This might take some time, please, be patient.", 'pod setup', false)
+install("six", "'six' python package... This might take some time, please, be patient.", 'pip install six', false)
```

Comment:
If I'm not mistaken, the `pip` is not installed by default on macOS, maybe we have to install it first?